content/en/docs/concepts/scheduling-eviction/eviction-policy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Eviction Policy
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page is an overview of the Eviction Policy used by `kubelet`.
```

Comment:
nit (OK to merge without fixing): `kubelet` is usually written without backticks.

On the wording, I suggest:
```suggestion
This page is an overview of Kubernetes' policy for eviction.
```